web/crux-ui/src/components/products/versions/images/image-config-filters.tsx

```diff
@@ -0,0 +1,149 @@
+import { DyoLabel } from '@app/elements/dyo-label'
+import { ImageConfigFilterType, IMAGE_CONFIG_FILTERS } from '@app/models'
+import clsx from 'clsx'
+import useTranslation from 'next-translate/useTranslation'
+import { useEffect, useState } from 'react'
+
+type BaseImageConfigFilterType = 'all' | 'common' | 'docker' | 'kubernetes'
+type FilterSet = Record<BaseImageConfigFilterType, ImageConfigFilterType[]>
+
+export const filterSet: FilterSet = {
+  all: [],
+  common: [
+    'name',
+    'environments',
+    'secrets',
+    'ingress',
+    'expose',
+    'user',
+    'tty',
+    'importContainer',
+    'configContainer',
+    'ports',
+    'portRanges',
+    'volumes',
+    'commands',
+    'args',
+    'initContainers',
+  ],
+  kubernetes: [
+    'deploymentStrategy',
+    'customHeaders',
+    'proxyHeaders',
+    'loadBalancer',
+    'healthCheckConfig',
+    'resourceConfig',
+  ],
+  docker: ['logConfig', 'restartPolicy', 'networkMode', 'networks'],
+}
+
+interface ImageConfigFilterProps {
+  onChange: (filters: ImageConfigFilterType[]) => void
+}
+
+const ImageConfigFilters = (props: ImageConfigFilterProps) => {
+  const { onChange } = props
+
+  const { t } = useTranslation('container')
+
+  const [filters, setFilters] = useState<ImageConfigFilterType[]>(
+    IMAGE_CONFIG_FILTERS as any as ImageConfigFilterType[],
+  )
+
+  const onBaseFilterChanged = (value: BaseImageConfigFilterType) => {
+    const filtersByBase = value !== 'all' ? filterSet[value] : IMAGE_CONFIG_FILTERS
+    const select = filters.filter(it => filtersByBase.indexOf(it) !== -1).length === filtersByBase.length
+
+    if (select) {
+      setFilters(filters.filter(it => filtersByBase.indexOf(it) === -1))
+    } else {
+      setFilters([...filters, ...filtersByBase.filter(it => filters.indexOf(it) === -1)])
+    }
+  }
+
+  const onFilterChanged = (value: ImageConfigFilterType) => {
+    const newFilters = filters.indexOf(value) !== -1 ? filters.filter(it => it !== value) : [...filters, value]
+    setFilters(newFilters)
+  }
+
+  useEffect(() => onChange(filters), [filters, onChange])
+
+  // Render
+
+  const getBorderColor = (type: BaseImageConfigFilterType): string => {
+    switch (type) {
+      case 'common':
+        return 'border-orange-400/50'
```

Comment:
Would be better if we predefine these colors in tailwind config like `border-dagent` and so on... No?